package.json

```diff
@@ -247,10 +247,10 @@
     "lint": "npx eslint 'src/**/*.ts'",
     "lint:fix": "npx eslint --fix 'src/**/*.ts'",
     "vscode:uninstall": "node ./out/uninstall",
-    "itly:verify": "itly verify -u --skip-update-on-default-branch"
+    "ampli:verify": "ampli status -u --skip-update-on-default-branch"
```

Comment:
@michelkaporin , would it not be consistent to have `ampli:status` command to match the real command?

Reply:
"ampli:verify" is more expressive in what command is doing than "ampli:status". I am frankly not sure why they've renamed the command to status, since [their docs](https://developers.data.amplitude.com/using-the-ampli-cli#step-5-verify-the-instrumentation) talk about "verification" still.

The naming here is consistent with the npm command in registry too, so I'd prefer to keep it.